### ISSUE TITLE
error in autogen.sh with paths with spaces 

### ISSUE THREAD

**Issue by mondaugen** [libsndfile/libsndfile#350]
```
bash-3.2$ ./autogen.sh 
checking for autogen ... yes
checking for autoconf ... yes
checking for automake ... yes
checking for libtool ... glibtoolize
checking for pkg-config ... yes
checking for python ... yes
Generating configuration files for libsndfile, please wait ... 
  aclocal 
  glibtoolize --automake --force
  autoheader
  automake --add-missing 
configure.ac:58: installing 'build-aux/ar-lib'
configure.ac:38: installing 'build-aux/compile'
configure.ac:28: installing 'build-aux/config.guess'
configure.ac:28: installing 'build-aux/config.sub'
configure.ac:33: installing 'build-aux/install-sh'
configure.ac:33: installing 'build-aux/missing'
Makefile.am: installing 'build-aux/depcomp'
  autoconf
./autogen.sh: line 164: cd: /tmp/test: No such file or directory

Installing git pre-commit hook for this project.

bash-3.2$ pwd
/tmp/test w spaces/libsndfile
```

Solution is to put `""` around `$olddir` in `cd $olddir`. Will submit patch shortly.

**Comment by SoapGentoo** [comment]
paths with spaces are a lost cause with Make. Make is inherently not whitespace-in-paths safe, so trying to fix this is mostly a pointless endeavour.

**Comment by mgeier** [comment]
I think this issue is obsolete since `autogen.sh` has been removed (and using `autoreconf -vif` has been recommended instead).

I just tried it on Linux and was able to build `libsndfile` this way, even though the source code was located in a directory containing spaces.

There still seems to be an issue with `libtool` on macOS though: https://github.com/autotools-mirror/libtool/pull/4, but this is unrelated to `libsndfile`.
